### DISCUSSION TITLE
Schedules Direct: Lost channels after lineup change

### DISCUSSION THREAD

**Discussion by CptKirk-1701** [OpenSageTV/Community#68] [Guide/EPG]
Schedules Direct Houston lineup changed from "Comcast - Digital - Houston" to "Xfinity - Digital - Houston" so had to redo all channel scans after removing the old lineup and changing to the new lineup.

When I log into my Schedules Direct account it shows as:

Name: Xfinity - Digital
LineupID: USA-TX62678-X
Type: SD-JSON

Since I had done that I have found a few channels so far that used to show up in SageTV with SD but now no longer show up.

1110 Fox News HD
1111 CNN HD
1456 Lifetime Movie Network HD

I did a new channel scan in the HDHomeRun Setup app (set to JRiver Media Center for BDA Compatibility Mode) and applied it.  Those channels are found and play in the HDHomeRun Setup app.

After doing the scan in the HDHomeRun Setup app, I then did a new FULL channel scan in SageTV.

Those channels were still not found.

Those are the only channels that I have noticed so far with problems.

I still have a Tivo in the house and it also finds and plays those channels with no problems.

Trying to figure out where the disconnect is in SageTV and Schedules Direct.

I use https://tvlistings.zap2it.com/?aid=gapzap, with Xfinity - Digital (77055) selected as the provider to check available channels.

I will also post over there.

**Comment by CptKirk-1701** [comment]
Schedule Direct got back to me and said that those three channels exist in the lineup on their side.

> They still exist on the lineup on our side:
> 1110: FNCHD "Fox News Channel HD" (stationID 60179)
> 1111: CNNHD "CNN HD" (stationID 58646)
> 1456: LMNHD "LMN HD" (stationID 55887)

So I guess I need to figure out what is going on in SageTV.

**Comment by Ugadata** [comment]
I am presuming you have a Prime Cable Card device.  You can use the HDHomeRun software or you open a browser and enter the IP address of the device.  Either way, there is a section that shows/lists the channels/stations that the device can see.  You can use this to verify that the device can see those channels (I suspect the channels are there but it needs to be verified).
F.W.I.W.  - There are channels that are part of the lineup but cannot be accessed by a cable card.  These stations are using an IPTV format (so not a DRM issue). 

I have Comcast/Xfinity and many of the HD  channels can be referenced using 3 or 4 digits for the channel number.  Maybe the channels are there but not where you were expecting them to be?

Maybe somehow, someway the channels have been disabled in Sage so they won't show up in the guide?

I suspect you may have already verified some of the above, but it is also easy to overlook when these kind of issues are happening.

**Comment by CptKirk-1701** [comment]
I mentioned in the post that I used the 'HDHomeRun Setup app' to scan and find the channels.

I can also use 'HDHomeRun Setup app' to launch the channels in VLC and watch them.

I just checked the HDHR Prime using the IP address and the channels mentioned are also shown there.

I am currently using the lower 3 digit channel numbers in SageTV so I can still watch the channels.

> 638 Fox News HD
> 625 CNN HD
> 671 Lifetime Movie Network HD

But it is just strange that the 4 digit channels numbers for those channels just are not seen/discovered in SageTV.

**Comment by andyvshr** [comment]
Is this the issue: SageTV lists each channel once in the guide. If different lineups have the same exact channels listed under different numbers, you will see them in the guide using only 1 of the logical numbers, SageTV tunes each channel using its actual physical number for whatever tuner it is using for the channel when recording something.

**Reply by CptKirk-1701** [reply]
Nope, SageTV has been finding and displaying all the 3 digit channels number and also all the 4 digit channel numbers for a long time.

This is just my first time of noticing that SageTV is not finding/showing just a few of the 4 digit channel numbers.

**Comment by Ugadata** [comment]
Hmm, interesting.  I just re-did my cable lineup this past weekend from 4 digit back to 3 digit tuning.

Friday night all was good using 4 digit tuning.  Saturday some cable channels getting "No Signal" using the 4 digits for tuning.  Tried the 3 digits and it worked, so all cable channels went back to 3 digit.  Consistency .

**Comment by Ugadata** [comment]
Comcast/Xfinity actual has both a 3 digit and 4 digit "physical" channel for many of the channels/stations on the HD lineup.   Typically you can use either (3 or 4 digits) for tuning.  And yes, you could set the logical channel with the 4 digits and the physical channel with 3 digits (and vice-versa).

**Reply by andyvshr** [reply]
Are the station IDs in the lineup different?

**Comment by dmasters74** [comment]
Hi. 

Can I ask is lineup USA-TX62678-X the one with missing epg data?

Also do you have SageX plugin installed?  I will post a SageX command that will extract your Lineups in Sage.

Once I have that I can compare the lineup from what is listed in USA-TX62678-X on the SD servers. 

Dean

**Comment by CptKirk-1701** [comment]
USA-TX62678-X – Yes

 

Plugin – there are two of them (screenshot attached).

 

Installed -            SageTV Sagex API Extensions

Not Installed -    SageTV Sagex Remote API Extensions

 

Thanks,

Kirk

**Reply by dmasters74** [reply]
Sorry for the delay in getting back to you. 

I added USA-TX62678-X to my feed and found the following 

60179	Fox News Channel HD
58646	CNN HD
10918	Lifetime
60150	Lifetime HD

So they are there. If you are saying you can view them in HDHomerun then I guess you going to have to manually remap the channels. Like how the rest of us outside of the USA have to lol

If you go into your lineups and click on a channel > Remap this Channel. Do the above show channels show up?

**Reply by andyvshr** [reply]
Again: if the 3-digit and 4-digit channels for something like CNN-HD have the same ID, 58646, it will be listed once in the guide.

This was a long-time point of confusion for many in the old forum. Even if the channels are on another lineup with the same ID, they will still end up in the guide under a single channel number, not twice.

My comment is based an above post containing:
>> I am currently using the lower 3 digit channel numbers in SageTV so I can still watch the channels.

>> 638 Fox News HD
>> 625 CNN HD
>> 671 Lifetime Movie Network HD

>> But it is just strange that the 4 digit channels numbers for those channels just are not seen/discovered in SageTV.

^^^ This makes it sound like the channels are still on that tuner, but being shown in the guide using the 3-digit listing.

If you want them to use 4 digits, remap the logical channel number, or even disable the 3-digit versions in the channel lineup.

**Comment by CptKirk-1701** [comment]
GAH!

Well, sidenote off topic.  I got that via gmail thinking it was an actual email from dmasters74 and just replied to it.  Not thinking/knowing that the email was generated by github and would feed my email reply back into the forum.

REALLY do not like that.  Is there a way to turn that off?  Going to start looking.
